### ISSUE TITLE
What's the future for keys.pub?

### ISSUE THREAD

**Issue by lessless** [keys-pub/keys#168]
Hi folks,

The latest commit dates back to 2021 and that makes me wonder if there is any future for keys.pub? 

Regards.

**Comment by gabriel** [comment]
I've moved onto other projects for the moment. I'm not sure if I'll come back to this one.

**Comment by austinjp** [comment]
Oh man. Yikes. I really love keys.pub and moved to it for key management after keybase was acquired.

@gabriel your work on this is inspirational. Could you explain why you're unsure about returning to this project? Genuine question, I'm sure people would like to understand and help. Scoping the issue would be useful. Is it about time+money, or technical issues, or...?

Would it make sense to start some sort of team to support keys.pub into the future?

I'd really love for keys.pub to continue and grow.

**Comment by svoop** [comment]
@austinjp For online identity proofs, you might want to check out [Keyoxide](https://keyoxide.org) (I'm not affiliated in any way). It's headless, no central server is necessary. It's basically a convention to store claims at predefined places on services like GitHub or Mastodon, then store the corresponding proofs in your GPG-key as notation.

It's then possible to verify the claim against the proof using GPG alone. This connects different online accounts on different services to one GPG key – and by extension online identity. This process is not simple, which is why simple clients exist for the web, Android etc. They basically consume a GPG-key fingerprint, collect the proofs and use them to verify the claims. The result is then rendered e.g. as a webpage which lists all services and their proof status (red or green). You can link to this public webpage e.g. from your own homepage.

There's also a keyoxide CLI tool to simplify your proof-management.

It's no doubt more difficult than Keybase or keys.pub, but the fact that no central server is required makes sure that Zoom can't acquire it nor will it just day when developers like @gabriel take the call to move on – which is totally understandable, we've all been there numerous times.

**Comment by austinjp** [comment]
Thanks @svoop Yes, Keyoxide was on my list of options to check out. Looks good so far.

**Comment by TaaviE** [comment]
Maybe this issue could be pinned so that people could see it first on GitHub?

**Comment by pataquets** [comment]
Agreed on pinning the issue, though under a more meaningful title, which would depend on @gabriel intentions. Be it a "co-maintainers/help wanted" or "looking for adoption", whatever @gabriel feels best.
It would be sad to see such a useful project languish.
Of course, priorities changes, as life does and there are a lot of caveats in project maintainership, but letting the project die would be the worst scenario.
Some links I've thought might be worth a read:
- https://opensource.guide/best-practices/
- https://medium.com/@shazow/how-to-hand-over-an-open-source-project-to-a-new-maintainer-db433aaf57e8

Also dropping https://adoptoposs.org which might come handy.

Whatever comes, also a big thank you to @gabriel for the great work and for sharing it!

**Comment by gabriel** [comment]
Thanks for the kind words. I'm don't know what to do with this project, I kind of want to resurrect it but my financial and mental health have to come first. Maybe someone will fork this or copy some ideas from it. But it should at least have warnings so people don't have expectations.

I've pinned this issue. I added this to the project README.

:warning: **Unfortunately, this project is not currently being maintained or worked on.** :warning:

Also on the website, a warning about the project not being worked on.

**Comment by pataquets** [comment]
So sad to hear that, @gabriel. It's very thoughtful on your part to clarify expectations, thanks a lot.
I hope your personal issues improve. First things first.
Reading the website, I understand that when things improve (hope it happens sooner than later) and you feel like it (which might not happen and it'll be okay), if you get some sponsorship (or maybe some help) you might eventually pick up again, which is perfectly fine as expectations go. However, I've noticed the message on the website varies slightly with the one in the README. I've sent PR #171 to match them (since I think website's is more useful and might help better). Also I've linked to this issue to provide better context.

To those interested in the project or picking up/helping/forking, be welcome to chime in!

Again, thanks @gabriel for sharing and I wish you the best and get well soon. Take care.
